wikitextprocessor/dumpparser.py

```diff
@@ -5,13 +5,15 @@
 import hashlib
 import logging
 import os
+from psutil import disk_partitions
```

Comment:
This would get import error unless package `psutil` is added as dependency to the` pyproject.toml` file.

Reply:
Adding psutil is as a dependency is not optimal, but should be fine... I'm surprised the Python standard libraries don't have something for this.